Tests/BugsnagClientTests.m

```diff
@@ -289,4 +291,9 @@ - (NSDictionary *)generateLargeMetadata {
     return dict;
 }
 
+- (void)testBSSerializeJSONDictionary {
+    char* dest = NULL;
+    BSSerializeJSONDictionary(@{@1: @"a"}, &dest);
```

Comment:
This test doesn't assert anything?

Reply:
It's just making sure no exceptions bubble up.

Reply:
ah! makes sense. that could probably use a comment for future travelers.